src/components/Layout/Sidebar.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Avatar from './Avatar';
+import { UserProfile } from 'services/user';
+
+import styles from './Layout.module.css';
+
+interface ComponentProps {
+  show: boolean;
+  setShow: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
Should we import `Dispatch` and `SetStateAction` and use them here?
```suggestion
  setShow: Dispatch<SetStateAction<boolean>>;
```

Reply:
Updated in [b759945](https://github.com/hanam1ni/react-surveyor/pull/60/commits/b759945159bbd42134f9e36837fbe450683ac6e9)